kong-0.14.0-0.rockspec

```diff
@@ -195,6 +195,10 @@ build = {
     ["kong.plugins.oauth2.daos"] = "kong/plugins/oauth2/daos.lua",
     ["kong.plugins.oauth2.api"] = "kong/plugins/oauth2/api.lua",
 
+    ["kong.plugins.log-buffering.json_serializer"] = "kong/plugins/log-buffering/json_serializer.lua",
```

Comment:
The naming is at odds with other previous pieces, and is somewhat confusing.

Since the `json_serializer` and `lua_serializer` both are called "serializer", that makes me think that they have something to do with `kong.plugins.log-serializers.basic`. (But then, why the differences in location and names)

Upon looking at their source code, I see that they are completely different. I would recommend finding a different word than "serializer" for them (maybe "producers"?)

Reply:
I reused the naming of the Galileo log buffering module, which used `serializers`. I agree it is confusing with `log-serializers`. Thanks for the suggestion, will switch to "producers".